draft-ietf-quic-multipath.md

```diff
@@ -461,60 +460,35 @@ connectivity or local preferences. After an endpoint abandons
 a path, the peer can expect to not receive any more non-probing packets on
 that path.
 
-An endpoint that wants to close a path SHOULD explicitly
-terminate the path by sending a PATH_ABANDON frame (see
-{{path-abandon-close}}). Note that while abandoning a path will cause
+An endpoint that wants to close a path MUST explicitly
+terminate the path by sending a PATH_ABANDON frame.
+Note that while abandoning a path will cause
 connection ID retirement, the inverse is not true: retiring the associated connection IDs
 does not indicate path abandonment (see further {{consume-retire-cid}}).
-Implicit signals such as idle time or packet losses might be
-the only way for an endpoint to detect path closure (see {(idle-time-close}})
-if connectivity is broken on that path.
-
-Note that other explicit closing mechanisms of {{QUIC-TRANSPORT}} still
-apply on the whole connection. In particular, the reception of either a
-CONNECTION_CLOSE ({{Section 10.2 of QUIC-TRANSPORT}}) or a Stateless
-Reset ({{Section 10.3 of QUIC-TRANSPORT}}) closes the connection.
```

Comment:
@huitema why did you remove this paragraph?

Reply:
Can't remember why. Restore it if you want.